test/test-helper.js

```diff
@@ -817,19 +818,33 @@ Ccm.prototype.waitForUp = function (callback) {
   var retryCount = 0;
   var self = this;
   utils.whilst(function () {
-    return !started && retryCount < 10;
+    return !started && retryCount < 60;
   }, function iterator (next) {
-    self.exec(['node1', 'showlog'], function (err, info) {
-      if (err) {
-        return next(err);
-      }
-      var regex = /Starting listening for CQL clients/mi;
-      started = regex.test(info.stdout.join(''));
+    var nodesTested = 0;
+    var ping = true;
+    utils.whilst(function () {
+      return nodesTested < self.nodeLength;
+    }, function pingNodes (nextIp) {
+      var pingIp = '127.0.0.' + (nodesTested + 1);
+      nodesTested++;
```

Comment:
We could include in the trace that an attempt to connect to `pingIp:9042` is being made, ie: `helper.trace(...)`